In [1]:
import pandas as pd

# Loading the data from the provided CSV file
file_path = 'C:/Users/Harvey/Downloads/Eye Movements.csv'
eye_movements_data = pd.read_csv(file_path)

# Displaying the first few rows of the data to understand its structure
eye_movements_data.head()


ParticipantNumber   Condition  CriticalRecall
0                  1  Horizontal               4
1                  3    Fixation              14
2                  4  Horizontal              12
3                  6    Fixation               4
4                  7  Horizontal              11

## Assumption 1: Level of Measurement
The dependent variable, 'CriticalRecall', is measured at a ratio level, satisfying this assumption.

## Assumption 2: Random Sampling
It is assumed that the data was collected from a random sample.

## Assumption 3: Independence of Observations
Each participant's recall ability is assumed to be independent of others.

## Assumption 4: Normality Test

We use the Shapiro-Wilk test to check for normality in each group.

In [2]:
import pandas as pd
from scipy.stats import shapiro, levene, ttest_ind

fixation_group = eye_movements_data[eye_movements_data['Condition'] == 'Fixation']['CriticalRecall']
horizontal_group = eye_movements_data[eye_movements_data['Condition'] == 'Horizontal']['CriticalRecall']

# Normality Test
normality_fixation = shapiro(fixation_group)
normality_horizontal = shapiro(horizontal_group)
normality_fixation, normality_horizontal


(ShapiroResult(statistic=0.9260098338127136, pvalue=0.07941146194934845),
 ShapiroResult(statistic=0.9590799808502197, pvalue=0.39645466208457947))

## Assumption 5: Homogeneity of Variances
Levene's test is used to assess the equality of variances.

In [3]:
# Levene's Test for Equal Variances
equal_variances_test = levene(fixation_group, horizontal_group)
equal_variances_test


LeveneResult(statistic=7.504307034421693, pvalue=0.008670566634332692)

## Independent Samples T-Test
Given the violation of the homogeneity of variances, we use Welch's t-test.

In [4]:
# Welch's T-Test
t_test_result = ttest_ind(fixation_group, horizontal_group, equal_var=False)
t_test_result


Ttest_indResult(statistic=2.8234133654901394, pvalue=0.007351503583719712)

Test Statistic (t-value): 2.823
P-value: 0.0074

### Short Report

Results: The test indicated a significant difference in recall performance between the two groups (t = 2.823, p = 0.0074). Participants in the horizontal eye movement condition exhibited different recall abilities compared to those in the fixed gaze condition.

Conclusion: The findings suggest that horizontal eye movements have a significant impact on the memory retrieval process in comparison to a fixed gaze. This aligns with the hypothesis that eye movements can influence cognitive processes such as memory recall.